Core/Core/Models/GraphTraversal/DefaultTraversal.cs

```diff
@@ -21,7 +24,6 @@ public static GraphTraversal CreateTraverseFunc(ISpeckleConverter converter)
       .ContinueTraversing(b =>
       {
         var membersToTraverse = b.GetDynamicMembers()
-          .Concat(displayValueAliases)
```

Comment:
Removing this from the general default traversal - was it here to address what was intended for Revit but isn't valid for others?

Was this the QGIS data being replicated issue?